cve-2021-25281.py

```diff
@@ -22,6 +22,7 @@
 from pathlib import Path
 from secrets import token_hex
 from textwrap import dedent
+from simplejson import JSONDecodeError
```

Comment:
```suggestion
from json import JSONDecodeError
```
Python's native `json` package [provides](https://docs.python.org/3/library/json.html#json.JSONDecodeError) `JSONDecodeError`. There's no need to import it from a third-party package.